starters/lit-mobx-vaadin/README.md

```diff
@@ -0,0 +1,77 @@
+## Lit-Mobx-Vaadin Starter Kit
+
+This starter kit features Lit, Mobx and Vaadin elements.
+
+## Overview
+
+### Tech Stack
+
+- [Lit](https://lit.dev/) - JavaScript framework for writing web components
+- [Mobx](https://mobx.js.org/) - State management library
+- [Vaadin](https://vaadin.com/components) - Routing and UI components
+
+### Included Tooling
+
+- [Playwright](https://playwright.dev/) - Testing library
+- [TypeScript](https://www.typescriptlang.org/) - Type checking
+- [Storybook](https://storybook.js.org/) - Component library
+- [ESLint](https://eslint.org/) - Code linting
+- [Prettier](https://prettier.io/) - Code formatting
+
+## Getting Started
+
+### Prerequisites
+* Node.js 16.8 or later installed
+
+### Example Components
+
+In the `starters/lit-mobx-vaadin/src` directory you will find a few directories organizing the project. Files are organized in directories under `src` as such:
+
+- `api` - contains functions that can be used to contact the Starter.dev API
+- `components` - contains re-usable components used inside of the project
+- `pages` - contains page components that are rendered when certain routes are visited
+- `styles` - contains stylesheets used by multiple components
+
+### Development
```

Comment:
I would recommend moving this section above the "Example Components" one, just to get that starter script closer to the top.